includes/class-sensei-course.php

```diff
@@ -2779,6 +2780,50 @@ public static function get_courses_page_url() {
 		return apply_filters( 'sensei_course_archive_page_url', $course_page_url );
 	}
 
+	/**
+	 * Get the link to the course completed page.
+	 *
+	 * @since 3.13.0
+	 * @param int $course_id Course ID to append to URL.
+	 *
+	 * @return string The URL or empty string if page is not set or does not exist.
+	 */
+	public static function get_course_completed_page_url( $course_id = 0 ) {
+		$page_id = isset( Sensei()->settings->settings['course_completed_page'] ) ? intval( Sensei()->settings->settings['course_completed_page'] ) : 0;
+		$url     = $page_id ? get_permalink( $page_id ) : '';
+
+		if ( $url && 0 < $course_id ) {
```

Comment:
I think we could add a new check here just in case the page is removed to return the empty string instead of `false` that would return from `get_permalink`.

Maybe something like this instead of the `if`?

```php
$url = $url && 0 < $course_id ? add_query_arg( 'course_id', $course_id, $url ) : '';
```

Reply:
Fixed in https://github.com/Automattic/sensei/pull/4270/commits/3631aa24d46bd0936f4033cd54ff290dee19882c.